src/extensions/mountainview/MVSortingView/PreloadCheck.tsx

```diff
@@ -0,0 +1,115 @@
+import { Dialog } from '@material-ui/core';
```

Comment:
Here's the new PreloadCheck component -- it does the preloading and updates the props of the child appropriately